Engine.UnitTests/TestPlanCompositeRunTests.cs

```diff
@@ -114,14 +114,51 @@ public void ShortTimeSpanTest()
             var s = new[] { "10 ms", "1 ms", "100 us", "10 μs", "1 μs", "1 ns", "0 s" };
 
             for (int i = 0; i < v.Length; i++)
-            {
+            {   
                 var s1 = ShortTimeSpan.FromSeconds(v[i]);
                 var s2 = ShortTimeSpan.FromString(s[i]);
                 StringAssert.AreEqualIgnoringCase(s1.ToString(), s2.ToString());
 
             }
         }
 
+        [Test]
+        public void LongTimeSpanTest()
+        {
+            var seconds = new TimeSpan[] 
+            { 
+                new TimeSpan(0,0, 86400), 
+                new TimeSpan(0, 0, 84600) , 
+                new TimeSpan(0, 0, 2031720), 
+                new TimeSpan(0,0, 86700), 
+                new TimeSpan(0,0, 18003),
+                new TimeSpan(0,0, 260470),
+                new TimeSpan(0,0, 86399999),
+                new TimeSpan(0,0, 86459),
+                new TimeSpan(0,0, 1),
+                new TimeSpan(0,0, 0)
+            };
```

Comment:
Instead of these seconds, can we use the more specific way of declaring the time?
```
                new TimeSpan(1,0, 0, 0), 
                new TimeSpan(0, 23, 30, 0) , 
                new TimeSpan(23, 12, 22, 0), 
                new TimeSpan(1,0, 5,0),  ...
```